src/main/java/teammates/storage/entity/FeedbackSession.java

```diff
@@ -138,6 +152,19 @@ public FeedbackSession(String feedbackSessionName, String courseId,
         this.respondingStudentList = studentList == null ? new HashSet<String>() : studentList;
     }
 
+    @OnLoad
+    @SuppressWarnings("unused") // called by Objectify
+    private void populateUtcFieldsFromLegacyFieldsIfRequired() {
+        if (startTime == null && endTime == null && sessionVisibleFromTime == null && resultsVisibleFromTime == null) {
```

Comment:
Can consider make the checking and setting of each field separate. While unlikely to happen, if not all of the above are null, there may be some unwanted consequences in the setting of the `*Utc` fields.
```java
if (startTime == null) {
    startTimeUtc = ...;
}

// do the same for the other fields
```

Reply:
If not all of them are null, it indicates that the entity has not been migrated yet. The 4 fields are annotated with `@IgnoreSave`, so they will all become null as long as the entity is saved with the new code.